### ISSUE TITLE
Upload `objects.inv` for intersphinx

### ISSUE THREAD

**Issue by prisae** [simpeg/simpeg#764]
Looks like at the moment the `objects.inv`-sphinx file is not uploaded to the docs. This file is necessary to do intersphinx-mappings, which is a great tool.

**Comment by prisae** [comment]
https://discretize.readthedocs.io/en/stable and now http://discretize.simpeg.xyz/en/master do have the `objects.inv`, so just SimPEG is missing.

Is there something similar to the readthedocs `en/stable` to point to the latest stable release instead of master with the new discretize documentation?

**Comment by lheagy** [comment]
Right now, master and the latest release of discretize should be in sync. Each time a new pr is merged, we create a new release. So for now having one should be sufficient. Did you have something else in mind @prisae?

As for SimPEG `objects.inv` thanks for the reminder. I will take another look at this. Eventually, I would prefer if we can move the SimPEG docs also onto github pages rather than google app engine, and this would streamline things. Perhaps if @banesullivan has some time at some point he might be willing to help?? 😉 (or provide some pointers!)

**Comment by prisae** [comment]
No, then it is fine. (I just wasn't sure if `master` is necessarily `stable`).

**Comment by prisae** [comment]
Pinging @jcapriot , because you fixed https://github.com/simpeg/simpeg/issues/857 :-D - it would be really cool to be able to use intersphinx on SimPEG.
